alexrallen/sonic-buildimage
# [device][mellanox] Add 2x40G support to MSN4700 platform

#### Why I did it

MSN4700 platform has 8 lanes per port and thus can support 2x40G with each lane running at 10G

#### How I did it

Added 40G to 2x200G breakout mode in platform.json

#### How to verify it

Run `config int break Ethernet0 2x40G[200G,100G,50G,25G,10G,1G]`
And verify the command runs successfully and the port speed was set to 40G with a 2x breakout. 

#### Which release branch to backport (provide reason below if selected)


#### Description for the changelog
[device][mellanox] Add 2x40G support to MSN4700 platform


#### A picture of a cute animal (not mandatory but encouraged)
TBD
